src/lib/recipes.ts

```diff
@@ -422,6 +422,104 @@ const xiphosRecipes: IRecipe[] = [
   ),
 ];
 
+// HH-a "Javelin" half-truck recipes
+const javelinRecipes: IRecipe[] = [
+  // Garage production: 20 x Refined Materials → 1 x HH-a "Javelin"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 20 }],
+    [{ stuff: Vehicles.Javelin, count: 1 }]
+  ),
+  // Mass Production Factory: 150 x Refined Materials → 3 crates of 3 x HH-a "Javelin" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 150 }],
+    [{ stuff: Vehicles.Javelin, count: 9 }]
+  ),
+];
+
+// HH-b "Hoplite" half-truck recipes
+const hopliteRecipes: IRecipe[] = [
+  // Garage production: 22 x Refined Materials → 1 x HH-b "Hoplite"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 22 }],
+    [{ stuff: Vehicles.Hoplite, count: 1 }]
+  ),
+  // Mass Production Factory: 165 x Refined Materials → 3 crates of 3 x HH-b "Hoplite" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 165 }],
+    [{ stuff: Vehicles.Hoplite, count: 9 }]
+  ),
+];
+
+// HH-d "Peltast" half-truck recipes
+const peltastRecipes: IRecipe[] = [
+  // Garage production: 24 x Refined Materials → 1 x HH-d "Peltast"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 24 }],
+    [{ stuff: Vehicles.Peltast, count: 1 }]
+  ),
+  // Mass Production Factory: 180 x Refined Materials → 3 crates of 3 x HH-d "Peltast" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 180 }],
+    [{ stuff: Vehicles.Peltast, count: 9 }]
+  ),
+];
+
+// Niska Mk. I Gun Motor Carriage recipes
+const niskaMkIRecipes: IRecipe[] = [
+  // Garage production: 30 x Refined Materials → 1 x Niska Mk. I Gun Motor Carriage
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 30 }],
+    [{ stuff: Vehicles.NiskaMkI, count: 1 }]
+  ),
+  // Mass Production Factory: 225 x Refined Materials → 3 crates of 3 x Niska Mk. I Gun Motor Carriage (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 225 }],
+    [{ stuff: Vehicles.NiskaMkI, count: 9 }]
+  ),
+];
+
+// Niska Mk. II Blinder recipes
+const niskaMkIIRecipes: IRecipe[] = [
+  // Garage production: 32 x Refined Materials → 1 x Niska Mk. II Blinder
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 32 }],
+    [{ stuff: Vehicles.NiskaMkII, count: 1 }]
+  ),
+  // Mass Production Factory: 240 x Refined Materials → 3 crates of 3 x Niska Mk. II Blinder (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 240 }],
+    [{ stuff: Vehicles.NiskaMkII, count: 9 }]
+  ),
+];
+
+// Niska Mk. III Scar Twin recipes
+const niskaMkIIIRecipes: IRecipe[] = [
```

Comment:
Only one recipe. requirements for one MK3:
5 x [Processed Construction Materials](https://foxhole.wiki.gg/wiki/Processed_Construction_Materials) ProcessedConstructionMaterialsIcon.png
5 x [Assembly Materials IV](https://foxhole.wiki.gg/wiki/Assembly_Materials_IV) AssemblyMaterials4Icon.png
1x [Niska Mk. I Gun Motor Carriage](https://foxhole.wiki.gg/wiki/Niska_Mk._I_Gun_Motor_Carriage) HalfTrackWarVehicleIcon.png